ion/writer.go

```diff
@@ -78,6 +78,12 @@ type Writer interface {
 	// Annotations adds multiple annotations to the next value written.
 	Annotations(values ...string) error
 
+	// Annotation adds a single annotation as a SymbolToken to the next value written.
+	AnnotationAsSymbol(val SymbolToken) error
+
+	// Annotations adds multiple annotations as SymbolTokens to the next value written.
+	AnnotationsAsSymbols(values ...SymbolToken) error
```

Comment:
The documentation should indicate whether passing a `SymbolToken` that's not already in the table is an `error` or not.